src/test/java/tests/Tests.java

```diff
@@ -0,0 +1,47 @@
+package tests;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import staticdata.WebUrls;
+
+public class Tests extends BaseTest {
+    @Test
+    public void addRemove() {
```

Comment:
Please don't forget 'Test' in the name of test methods. And don't forget about making formatting before sending (ctrl+alt+l)